internal/backends/qt/qt_accessible.rs

```diff
@@ -317,6 +317,7 @@ cpp! {{
                     i_slint_core::items::AccessibleRole::Table => QAccessible_Role_Table,
                     i_slint_core::items::AccessibleRole::Tree => QAccessible_Role_Tree,
                     i_slint_core::items::AccessibleRole::TextInput => QAccessible_Role_EditableText,
+                    i_slint_core::items::AccessibleRole::Switch => QAccessible_Role_CheckBox,
```

Comment:
I didn't find a closer accessible role on the Qt side unfortunately.

Reply:
makes sense